apps/src/netsim/netsim.js

```diff
@@ -301,6 +303,15 @@ NetSim.prototype.shouldShowAnyTabs = function () {
 };
 
 /**
+ * @returns {boolean} TRUE if the "resetShard" flag is found in the URL
```

Comment:
Oops.  Should have used [test](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test).  Fixing.